doc/source/whatsnew/v0.19.0.rst

```diff
@@ -863,8 +863,8 @@ Furthermore:
 ``Period`` changes
 ^^^^^^^^^^^^^^^^^^
 
-``PeriodIndex`` now has ``period`` dtype
-""""""""""""""""""""""""""""""""""""""""
+The ``PeriodIndex`` now has ``Period`` dtype
```

Comment:
I think period is another one that will have to allow both period and Period. Can you use period lower case here please?